yt/frontends/gdf/tests/test_outputs.py

```diff
@@ -1,30 +1,28 @@
-from yt.frontends.gdf.api import GDFDataset
-from yt.testing import assert_equal, requires_file, units_override_check
-from yt.utilities.answer_testing.framework import (
-    data_dir_load,
-    requires_ds,
-    small_patch_amr,
-)
+import pytest
 
-_fields = ("density", "velocity_x")
+from yt.frontends.gdf.api import GDFDataset
+from yt.testing import requires_file, units_override_check
+from yt.utilities.answer_testing.answer_tests import small_patch_amr
 
+# Test data
 sedov = "sedov/sedov_tst_0004.h5"
 
 
-@requires_ds(sedov)
-def test_sedov_tunnel():
-    ds = data_dir_load(sedov)
-    assert_equal(str(ds), "sedov_tst_0004")
-    for test in small_patch_amr(ds, _fields):
-        test_sedov_tunnel.__name__ = test.description
-        yield test
-
+@pytest.mark.answer_test
+class TestGDF:
+    answer_file = None
+    saved_hashes = None
+    answer_version = "000"
 
-@requires_file(sedov)
-def test_GDFDataset():
-    assert isinstance(data_dir_load(sedov), GDFDataset)
+    @pytest.mark.usefixtures("hashing")
+    @pytest.mark.parametrize("ds", [sedov], indirect=True)
+    def test_sedov_tunnel(self, a, d, w, f, ds):
+        self.hashes.update(small_patch_amr(ds, f, w, a, d))
```

Comment:
With this test could the variables passed to the test be named more clearly?

Reply:
Fixed.